### PR TITLE
DEVEX-113 - Investigate: SDK on Promise fulfilled first, but the data…

### PR DESCRIPTION
**Pull request by osylko-bondtech**
Investigate: SDK on Promise fulfilled first, but the data takes a few seconds to load
https://bond-financial.atlassian.net/browse/DEVEX-113

**Reason**: Promise is returned when iframe rendered only. But there is async request inside iframe.
**Solution**: Add event listener to subscribe for message event. (This approach does not described in the vgs documentation)

### CODE REVIEW

File: dist/bond-sdk-cards.js.LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*!
```
Comment (dominicsardina) [inline dist/bond-sdk-cards.js.LICENSE.txt:1]: I never check these in since they're not our licenses.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bond-sdk-cards",
-  "version": "1.4.1",
+  "version": "1.4.2",
```
Comment (dominicsardina) [inline package.json:3]: Your new feature to delay the promise return isn't a bugfix. So I would use 1.5.0

File: src/bond-sdk-cards.js [hunk 3]
```diff
@@ -103,7 +103,14 @@ class BondCards {
     return new Promise((resolve, reject) => {
       const newIframe = this.internalShow.request(requestParams);
       if (newIframe) {
-        resolve(newIframe.render(htmlSelector, css));
+        const iframe = newIframe.render(htmlSelector, css);
+        const subscribe = (event) => {
+          if(event.data.messageName === "update" && event.data.payload.revealed === true){
```
Comment (dominicsardina) [inline src/bond-sdk-cards.js:108]: With the inconsistent spacing after "if" it seems like you don't have Prettier formatting your code in your IDE? We'd like all the devs to be consistent and we use Prettier and a customized set of AirBnb's ESLint rules to make sure everyone is doing the same syntax and code-style. Let me know if you'd like help setting those up. There's a confluence page with instructions too.

**Review by dominicsardina — CHANGES_REQUESTED** [review]

**Review by dominicsardina — APPROVED** [review]

**Comment by osylko-bondtech** [comment]
Update documentation SDK regarding to https://bond-financial.atlassian.net/browse/DEVEX-117

https://user-images.githubusercontent.com/78865935/114868743-eee26300-9dfe-11eb-9fe6-b1fede16c242.mov

**Review by dominicsardina — APPROVED** [review]

**Pull request merged**